overlay/files/etc/elemental/config.yaml

```diff
@@ -3,16 +3,16 @@ verify: false
 install:
   grub-entry-name: "kairos"
   system:
-    size: 2000
```

Comment:
Why increasing size for all flavors by default? This bumps partition size for all. 

Why not applying this only on Ubuntu?